### ISSUE TITLE
a2ensite error on debian8

### ISSUE THREAD

**Issue by Demonsthere** [pingworks/chef-dash#1]
Hey guys,
I have been using your cookbook for some time, and on my debian7 box it worked quite well, but after changing to debian8 this happens:
`bash[enable_apache_site] action run

```
       ================================================================================
       Error executing action `run` on resource 'bash[enable_apache_site]'
       ================================================================================

       Mixlib::ShellOut::ShellCommandFailed
       ------------------------------------
       Expected process to exit with [0], but received '1'
       ---- Begin output of "bash"  "/tmp/chef-script20160210-1181-d4epna" ----
       STDOUT: 
       STDERR: ERROR: Site dash-prod does not exist!
       ---- End output of "bash"  "/tmp/chef-script20160210-1181-d4epna" ----
       Ran "bash"  "/tmp/chef-script20160210-1181-d4epna" returned 1`
```

After some test it seem the problem is that the config file is called "dash-prod" and not "dash-prod.conf"

`vagrant@default-debian-8:/etc/apache2/sites-available$ sudo a2ensite
000-default.conf  dash-prod         default-ssl.conf
vagrant@default-debian-8:/etc/apache2/sites-available$ sudo a2ensite dash-prod
ERROR: Site dash-prod does not exist!
vagrant@default-debian-8:/etc/apache2/sites-available$ sudo mv dash-prod dash-prod.conf
vagrant@default-debian-8:/etc/apache2/sites-available$ sudo a2ensite dash-prod
Enabling site dash-prod.
To activate the new configuration, you need to run:
  service apache2 reload`

**Comment by cluk1** [comment]
Hi,

you are absolutely right. Debian 8 is shipping with apache 2.4 which needs the .conf suffix for the sites. The next problem is, that the auth module changed the syntax to allow access. 

We had to deal with both problems when we tried to use the cookbook on ubuntu-lts instead of debian. We just added a hack:

https://github.com/pingworks/chef-dash/blob/master/recipes/dash-prod.rb#L51

But the better way would probably be to decide on the apache version installed.
Could you prepare a pull request to fix this? ;)

**Comment by cluk1** [comment]
Should be fixed with version 0.5.0
